src/components/InstanceFilters/InstanceFilters.js

```diff
@@ -43,11 +63,160 @@ const InstanceFilters = props => {
       modesOfIssuance,
       natureOfContentTerms,
       tagsRecords,
+      query: { query = '', filters = '' },
+      onFetchFacets,
+      parentResources,
     },
     onChange,
     onClear,
   } = props;
 
+  const {
+    facets: { records = {} },
+  } = parentResources;
+
+  const intl = useIntl();
+  const [accordions, setAccordions] = useState({
+    effectiveLocation: true,
+    language: false,
+    resource: false,
```

Comment:
Same here, having the constants for facet names you can apply it as a calculated property names e.g.
[FACETS.TAGS]: true

Reply:
Thanks, changed.